### DISCUSSION TITLE
email-auth >> Message from server :: Server Key does not match

### DISCUSSION THREAD

**Discussion by Rai09Tejas** [saran-surya/email_auth#77] [Q&A]
everything is checked. Just gmail is not set to LESS SECURE APPS as the setting is unavailable as of now

**Comment by saran-surya** [comment]
Hi,
    I am currently looking for an alternative, will be updating as soon as
possible.

Thanks

**Comment by saran-surya** [comment]
### Kindy use test servers as of now, 

This should give you a good lead.
https://github.com/saran-surya/email_auth#update--google-turning-off-less-secure-apps--switching-to-oauth
